sdk/metric/meter.go

```diff
@@ -108,6 +122,9 @@ func (m *meter) Int64Histogram(name string, options ...metric.Int64HistogramOpti
 // It registers callbacks for each reader's pipeline.
 func (m *meter) int64ObservableInstrument(id Instrument, callbacks []metric.Int64Callback) (int64Observable, error) {
 	inst := newInt64Observable(m, id.Kind, id.Name, id.Description, id.Unit)
+	// If we are re-using the instrument, measures have already been appended
+	// and we don't need to append them again.
+	shouldAppendMeasures := len(inst.measures) == 0
```

Comment:
Won't this also apply if a view is applied to a different instrument?

Reply:
Not sure I understand.  This means either:

* This is the first time an instrument with that name/description/unit is created, thus the measures haven't been initialized yet, OR
* The instrument was previously created, but with a drop aggregation, so there weren't any measures. We will still have a drop aggregation, so in that case `shouldAppendMeasures` won't ever be checked below.